checkov/arm/checks/resource/AppServicePublicAccessDisabled.py

```diff
@@ -0,0 +1,22 @@
+from typing import Any
+
+from checkov.arm.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.enums import CheckCategories
+
+
+class AppServicePublicAccessDisabled(BaseResourceValueCheck):
+    def __init__(self) -> None:
+        name = "Ensure that Azure Web App public network access is disabled"
+        id = "CKV_AZURE_222"
+        supported_resources = ["Microsoft.Web/sites",]
```

Comment:
You can add the `Microsoft.Web/sites/slots` and `Microsoft.Web/sites/config` to the supported resource_types.